packages/guides/src/RenderContext.php

```diff
@@ -152,6 +152,24 @@ public function getCurrentFileName(): string
         return $this->currentFileName;
     }
 
+    /** @return string[] */
+    public function getCurrentFileRootline(): array
+    {
+        if ($this->getCurrentDocumentEntry() === null) {
+            throw new LogicException('Cannot get current document entry when not rendering a document');
+        }
+
+        $rootline = [];
+        $documentEntry = $this->getCurrentDocumentEntry();
+        $rootline[] = $documentEntry->getFile();
+        while ($documentEntry->getParent() instanceof DocumentEntryNode) {
+            $documentEntry = $documentEntry->getParent();
+            $rootline[] = $documentEntry->getFile();
+        }
+
+        return $rootline;
```

Comment:
As the render context is created per document, It would make sense to cache this rootline in context. That would reduce the impact of calling this method. Or should we store the rootline in the document entry?